smt/applications/mixed_integer.py

```diff
@@ -104,7 +104,8 @@ def __init__(
         self._cat_kernel_comps = cat_kernel_comps
         self._xtypes = xtypes
         self._xlimits = xlimits
-        self._surrogate.options["xlimits"] = self._xlimits
+        if "xlimits" in self._surrogate.options:
+            self._surrogate.options["xlimits"] = self._xlimits
```

Comment:
I am still puzzled by this `xlimits` option. Why not use the values themselves to normalize when using Gower distance? Like `standardization`?